### ISSUE TITLE
Only Loading on ISAPI DLL

### ISSUE THREAD

**Issue by kivervinicius** [danieleteti/delphimvcframework#37]
Hello, I'm doing a test to generate the dll, I generated the same but the post following his https://danieleteti.gitbooks.io/delphimvcframework/content/chapterdeploy_iis.html tips still does not run, it is only loading the page, did not nothing too much nor connection with bank or anything, the only thing I did was a render ('hello') based on the example of the github, if I do the same thing with the embedded server works.

Thank you.

**Comment by danieleteti** [comment]
Did you tried to create a simple webroker isapi? Does it work?

2016-04-11 23:51 GMT+02:00 Kiver Vinicius <notifications@github.com>:

> Hello, I'm doing a test to generate the dll, I generated the same but the
> post following his
> https://danieleteti.gitbooks.io/delphimvcframework/content/chapterdeploy_iis.html
> tips still does not run, it is only loading the page, did not nothing too
> much nor connection with bank or anything, the only thing I did was a
> render ('hello') based on the example of the github, if I do the same thing
> with the embedded server works.
>
> Thank you.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/danieleteti/delphimvcframework/issues/37>
>



-- 
*Daniele Teti*
CEO
bit Time Professionals
web: www.bittimeprofessionals.it <http://www.bittime.it>
blog: www.danieleteti.it
books: www.danieleteti.it/delphi-cookbook
mobile: +39 3496626822
office: +39 06 83393029

**Comment by kivervinicius** [comment]
Yes, the simple webbroker isapi, works perfectly well, i don't have idea how solve this.

**Comment by kivervinicius** [comment]
Now i am testing this with debug mode, and works.... on exit debug mode return a loading infinite

**Comment by kivervinicius** [comment]
in my tests, i discovery that my dll only work in debug mode, but the log not acused error in my programm, i'am confused

**Comment by danieleteti** [comment]
I'm investigating about iis DLL unloading. loggerpro will fix this.

**Comment by franciscpd** [comment]
Good day, the problem found is that when you add direct ISAPI permission for the generated DLL, does not carry the service, except when it is enabled the "Allow ISAPI modules not specified". I'm having this problem loading my service to cloud server.
